runtime/stout/src/constants.rs

```diff
@@ -16,8 +16,8 @@
 // limitations under the License.
 
 pub mod currency {
-	use kusama_runtime_constants as constants;
 	use polkadot_core_primitives::Balance;
+	use rococo_runtime_constants as constants;
```

Comment:
Same remark as for the Trappist runtime.

Reply:
Fixed in [88a0b6a](https://github.com/paritytech/trappist/pull/315/commits/88a0b6a5a8a246bf304a0f815d53f372c71b8323).